.changeset/tender-ravens-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': minor
```

Comment:
I think this should be major since it contains breaking changes

Reply:
I must have run the wrong command 🙃

Reply:
It's technically still a major change, but it feels so minor...

Reply:
I think it should still be major since it's breaking, and I think we probably need to bump the `resource-server.yaml` version as well. Thats how we handled it when I renamed a field here: https://github.com/interledger/open-payments/pull/270. Although the spec version bump was a minor bump 🤔... don't quite remember that reasoning.

On the topic of it feeling minor, Max pointed to some relevant blog posts and hackernews discussion in this comment https://github.com/interledger/open-payments/pull/270#discussion_r1250797466. I don't see any quick, easy wins there and think just doing a major bump still makes the most sense but it is interesting reading.